nrniv/Parallel/context.py

```diff
@@ -3,15 +3,15 @@
 pc = h.ParallelContext()
 
 def f(arg):
-  print '%d f(%s)'%(pc.id(), arg)
+  print(('%d f(%s)'%(pc.id(), arg)))
```

Comment:
Strange. When I use 2to3 it gives:
```
-  print '%d f(%s)'%(pc.id(), arg)
+  print('%d f(%s)'%(pc.id(), arg))
```

Reply:
My version is:
```
(venv_bare) savulesc@bbd-cjngk03:~/Workspace/nrntest/nrniv/Parallel$ pip show 2to3
Name: 2to3
Version: 1.0
Summary: Adds the 2to3 command directly to entry_points.
Home-page: UNKNOWN
Author: xoviat
Author-email: xoviat@gmail.com
License: MIT
Location: /home/savulesc/Workspace/venv_bare/lib/python3.8/site-packages
Requires: 
Required-by: 
```

Reply:
I see. In the end I manually patched the files